news/2019-07-11-project-loved-week-of-july-14th.md

```diff
@@ -0,0 +1,219 @@
+
+In case you don't read to the end of this post, be sure to submit suggestions for beatmaps to be added to Loved using the [Google form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)! All of the nominated beatmaps are chosen from submissions on this form, which you can view in the corresponding [Google sheet](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit). For osu!mania, please use [this Google form](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform) and [this Google sheet](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit) instead.
+
+### Navigation
+
+- [osu!standard](#osu)
+- [osu!taiko](#taiko)
+- [osu!catch](#catch)
+- [osu!mania](#mania)
+
+## <a name="osu" id="osu"></a>osu!standard
+
+osu!standard Loved candidates were chosen by [DigitalHypno](https://osu.ppy.sh/users/4384207), [-Roxas](https://osu.ppy.sh/users/1986262), [BeasttrollMC](https://osu.ppy.sh/users/3171691), [Bubbleman](https://osu.ppy.sh/users/5182050), [fartownik](https://osu.ppy.sh/users/56917) and [waywern2012](https://osu.ppy.sh/users/5870453)!
+
+---
+
+[![](/wiki/shared/news/2019-07-11-project-loved-week-of-july-14th/osu/1-bon-appetit-s-oldskool-happyhardcore-remix-short-ver.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[K A Z M A S A - Bon Appetit S (Oldskool HappyHardcore Remix) (Short Ver.)](https://osu.ppy.sh/beatmapsets/859515#osu) by [BarkingMadDog](https://osu.ppy.sh/users/3475189)  
+180 BPM, 3:13 | 5.87★ – 7.46★  
+*description written by [-Roxas](https://osu.ppy.sh/users/1986262)*
+
+Been looking for a good stream map to train your stamina? This is the map for you! BarkingMadDog's *Bon Appetit S* is mapped to varying tempos that start at 170 and go up to 220 BPM. Patterns in the map are mostly simplistic streams though have some accelerating ones throughout. This set also tests your finger control with triples and doubles into slider patterns so don't get caught off guard by them! This very popular map exceeds 200 favorites so it's definitely worth a shot!
+
+[![](/wiki/shared/news/2019-07-11-project-loved-week-of-july-14th/osu/2-uta.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Imperial Circus Dead Decadence - Uta](https://osu.ppy.sh/beatmapsets/469683#osu) by [Raikozen](https://osu.ppy.sh/users/3827077)  
+140 BPM, 8:48 | 7.71★  
+*description written by [-Roxas](https://osu.ppy.sh/users/1986262)*
+
+With Raikozen getting unbanned, you'll be glad to hear his maps are now available to appear in Loved voting! Starting off with his very popular Uta, it's also one of his more difficult maps sitting at 7.71 stars. The map consists of 8 minutes of huge AR10 jumps and hard spaced streams which makes falling off one of these patterns all that more deadly. However, if that's your type of map, then by all means drop this one a vote!
```

Comment:
Uh, I dont think is a good thing to point in news that an user is getting unbanned, and also praising that.

Reply:
Totally agree. At most you can say he's returned after a break, but this sentence is honestly not necessary either way.